source/index.js

```diff
@@ -2,15 +2,22 @@ var Pixi = require("pixi.js")
 var Afloop = require("afloop")
 var Keyb = require("keyb")
 
-var WIDTH = 230, HEIGHT = 130
+import Player from "./player.js"
 
-var renderer = Pixi.autoDetectRenderer(WIDTH, HEIGHT)
+window.GAME_WIDTH = 230
+window.GAME_HEIGHT = 130
```

Comment:
A Reference.js file was actually part of our formal design, this was just a kludge to get some functionality while I was trying to figure out JS imports. Moved to Reference.js in next commit.